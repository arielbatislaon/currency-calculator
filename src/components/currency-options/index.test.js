import React from 'react';
import renderer from 'react-test-renderer';

import CurrencyOptions from './index';

it('renders CurrencyOptions correctly ', () => {
  const tree = renderer.create(<CurrencyOptions />).toJSON();
  expect(tree).toMatchSnapshot();
});