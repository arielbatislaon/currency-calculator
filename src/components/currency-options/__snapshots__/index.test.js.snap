// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CurrencyOptions correctly  1`] = `
<div
  className="currency-option "
>
  
</div>
`;
